tests/test_distance.py

```diff
@@ -0,0 +1,92 @@
+import datetime
+import os
+import zipfile
+
+import numpy as np
+import pandas as pd
+import partridge as ptg
+from pyproj import Geod
```

Comment:
suggest adding as an import in the method so that only need it if it is called?

Reply:
Removed pyproj from tests for now. Eliminates it from project.